eos-toolbox-dev-depends

```diff
@@ -0,0 +1,74 @@
+# The eos-toolbox-dev package is intended to be included in the eos-toolbox
+# container images as a way to provide the base development packages needed
+# for common development.
+#
+# Packages listed here should be commonly used, of high value for
+# development, and expected to be relevant to a whole group of Endless
+# developers.
+#
+# Some packages included in the eos-dev package (e.g. gnome dev packages)
+# are not included here on purpose given they increase the container image
+# size quite substantially. Developers that need the extra packages can
+# still install them manually using apt-get from an eos-toolbox container.
+
+acl
+apt-file
+apt-transport-https
+apt-utils
+autoconf
+automake
+bash-completion
+buildah
+build-essential
+bzip2
```

Comment:
I think this is a transitive dependency of build-essential – could it be removed?

Reply:
ack, I will remove it explicitly but I am afraid of relying on other pkg deps in general but this one likely wont go away ever :).